icubam/backoffice/templates/base.html

```diff
@@ -115,6 +115,12 @@
               <p>Access Tokens</p>
             </a>
           </li>
+          <li class="nav-item">
+            <a href="/list_regions" class="nav-link">
```

Comment:
```suggestion
            <a href="/regions" class="nav-link">
```

Reply:
I think the fact that `GET /regions` returns a list of regions is expected and there is no need for `list` in the URL.

Reply:
as much as i agree with you, all the other handlers are the same + there it is slightly less error prone. But in principle i would prefer just regions.